.github/workflows/release.yaml

```
@@ -61,6 +66,8 @@ jobs:
         dbmode:
           - 'dbless'
           - 'postgres'
+        enterprise:
+          - 'yes'
```

Again, looks like we don't need to use a matrix here:

```suggestion
        enterprise:
          - 'yes'
```

---

The new matrix 
https://github.com/Kong/kubernetes-ingress-controller/pull/1758/files#diff-e426ed45842837026e10e66af23d9c7077e89eacbe6958ce7cb991130ad05adaR170-R175